### PR TITLE
CHKT-1425 Update iPay88 version

### PR DESCRIPTION
**Pull request by borisprimer**
We now have .xcframework instead of the library. Updated with the latest version downloaded from Google Drive URL.
JIRA: https://primerapi.atlassian.net/browse/CHKT-1425
Debug App PR: https://github.com/primer-io/primer-sdk-ios/pull/641

### CODE REVIEW

File: PrimerIPay88SDK/Classes/PrimerIPay88Payment.swift [hunk 1]
```diff
@@ -91,7 +91,7 @@ public class PrimerIPay88Payment: NSObject {
         iPay88Payment.prodDesc = self.prodDesc
         iPay88Payment.userName = self.userName
         iPay88Payment.userEmail = self.userEmail
-        iPay88Payment.userContact = self.userContact
+//        iPay88Payment.userContact = self.userContact
```
Comment (EvansPie) [inline PrimerIPay88SDK/Classes/PrimerIPay88Payment.swift:94]: Why is that? Is this needed?

File: PrimerIPay88SDK/lib/MH/IpayPayment.h [hunk 2]
```diff
@@ -13,7 +13,7 @@
     NSString *prodDesc;
     NSString *userName;
     NSString *userEmail;
-    NSString *userContact;
+//    NSString *userContact;
```
Comment (EvansPie) [inline PrimerIPay88SDK/lib/MH/IpayPayment.h:16]: Same as above

File: PrimerIPay88SDK/lib/MH/IpayPayment.h [hunk 3]
```diff
@@ -39,7 +39,7 @@
 @property (nonatomic, retain) NSString *prodDesc;
 @property (nonatomic, retain) NSString *userName;
 @property (nonatomic, retain) NSString *userEmail;
-@property (nonatomic, retain) NSString *userContact;
+//@property (nonatomic, retain) NSString *userContact;
```
Comment (EvansPie) [inline PrimerIPay88SDK/lib/MH/IpayPayment.h:42]: Same as above

File: PrimerIPay88MYSDK.podspec
Comment (EvansPie) [inline PrimerIPay88MYSDK.podspec:1]: Where are the `Package.swift` changes?

File: PrimerIPay88SDK/lib/MH/IpayPayment.h
Comment (EvansPie) [inline PrimerIPay88SDK/lib/MH/IpayPayment.h:1]: Why isn't the `/lib` folder removed?

File: PrimerIPay88SDK/lib/MH/IpayPayment.h (hunk 2 above)
Comment (borisprimer) [reply PrimerIPay88SDK/lib/MH/IpayPayment.h:16]: This is how the new iPay88 framework is deployed, this is one of the changes in their header files. I naturally commented out all the usages in our files.

File: PrimerIPay88SDK/lib/MH/IpayPayment.h
Comment (borisprimer) [reply PrimerIPay88SDK/lib/MH/IpayPayment.h:1]: Good point, removed.

File: PrimerIPay88MYSDK.podspec
Comment (borisprimer) [reply PrimerIPay88MYSDK.podspec:1]: I don't see Package.swift file anywhere in the repository... Lets do it together on a call maybe?

**Review by EvansPie — APPROVED** [review]

File: PrimerIPay88SDK/Frameworks/IpayPayment.h
[the remark this answers is not in the document]
Comment (borisprimer) [reply PrimerIPay88SDK/Frameworks/IpayPayment.h:1]: It is included in the downloaded iPay88.zip

File: PrimerIPay88SDK/Frameworks/Ipay.h
[the remark this answers is not in the document]
Comment (borisprimer) [reply PrimerIPay88SDK/Frameworks/Ipay.h:1]: It is included in the downloaded iPay88.zip

**Comment by borisprimer** [comment]
@xevious78 can you do the `pod trunk push` for the new version, I don't have permission. Should I merge this first back to master? @EvansPie

**Pull request merged**